### ISSUE TITLE
Compatibility with PhantomJS 2.0.0

### ISSUE THREAD

**Issue by nbauma109** [n1k0/casperjs#1150]
Hello,

I tried to run CasperJS with PhantomJS 2.0.0. I got the error message : "CasperJS needs PhantomJS v1.x".
I thought CasperJS would work just by removing the version check in the bootstrap.js.
But instead I got the error message:
"Couldn't find nor compute phantom.casperPath, exiting."
Could you have a look please ?

Kind regards,
Nicolas.

**Comment by christianhaller** [comment]
+1

**Comment by rdpanek** [comment]
You must read older issues before put a new issue ;-)

https://github.com/n1k0/casperjs/pull/1137

**Comment by goggelz** [comment]
insert these 4 lines into the bin/bootstrap.js file
	var system = require('system');
	var argsdeprecated = system.args;
	argsdeprecated.shift();
	phantom.args = argsdeprecated;
phantom.args is deprecated in the 2.0 version, this is the reason, why it fails

**Comment by n1k0** [comment]
@goggelz would you mind creating a new PR and mention me in its description? I don't have much time to work on casper these days, so any help is much appreciated. If we get a green build with Phantom2 added to the build matrix, I'll merge it asap. Thanks.

**Comment by DeividasK** [comment]
```javascript
(function (version) {
        // required version check
        if (version.major === 1) {
            if (version.minor < 8) {
                return __die('CasperJS needs at least PhantomJS v1.8 or later.');
            }
            if (version.minor === 8 && version.patch < 1) {
                return __die('CasperJS needs at least PhantomJS v1.8.1 or later.');
            }
        } else if (version.major === 2) {
            console.log("Warning PhantomJS v2.0 not yet released. There will not be any official support for any bugs until stable version is released!");
        }
        else return __die('CasperJS needs PhantomJS v1.x or v2.x');
    })(phantom.version);
```
I am confused. The code above is in the master branch. Is this the code that should be changed to accomodate v2? Because it seems that it already does.

**Comment by goggelz** [comment]
@nbauma109 @christianhaller 
#1154
should work fine for you

**Comment by n1k0** [comment]
Fixed by #1154.

**Comment by alexserver** [comment]
@goggelz thank you !
I've been searching for this args deprecating notice in phantomjs README, I swear I saw it, can you please share any useful link that talks about it ?
thanks
